homeworks/nagynorbert/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,25 @@
+# 2. homework - Nagy Norbert
+
+#user_inputs
+fict_name = input("Please type your character's name:").strip().capitalize()
+fict_age_days = 365 * int(input("Please add how many years old is your character:"))
+fict_experience = int(input("How many years experencie does your character have in Python programming?"))
+
+#extra
+answer = ("yes","no")
+fict_pro = input("Would your character be a pro Python developer? (Yes/No)").strip().lower()
+while fict_pro not in answer:
```

Comment:
Szia, már alakul, de így kétszer írod le ugyanazt a kérdést. Próbáld meg így:
```
while True:
    fict_pro = input("Would your character be a pro Python developer? (Yes/No)").strip().lower()
    if fict_pro in ["yes", "no"]:
        break
    else:
        print("Invalid input. Please answer with 'yes' or 'no'.")
```
Ezzel indítasz egy végtelen ciklust amiből kilépsz a break-kel ha "yes"-t vagy "no"-t válaszol. Holnap tanulunk ezekről bővebben, szóval ez tényleg csak egy nice to have dolog most.